Functions.Templates/Templates-v2/BlobTrigger-JavaScript/template.json

```diff
@@ -0,0 +1,47 @@
+{
+    "author": "Eric Jizba",
+    "name": "Azure Blob Storage trigger",
+    "description": "$BlobTrigger_description",
+    "programmingModel": "v4",
+    "language": "JavaScript",
+    "jobs": [
+        {
+            "name": "Create New Function",
+            "type": "CreateNewFile",

```

Comment:
Looking at the docs, there doesn't seem to be a supported `CreateNewFile` option, only `CreateNewApp` and `AppendToFile`. Am I missing something?

Also, are we not supporting the appending scenario?